.github/workflows/demo.yml

```diff
@@ -46,9 +46,12 @@ jobs:
       - name: "Allow some time for data modeling to finish syncing of deletions"
         run: |
           sleep 30
+      - name: "Clean the templates"
+        run: |
+          cdf-tk clean --env=demo ./build
```

Comment:
with the removal of `drop_data`, I assume we know want to clean the demo project before redeploying.

Reply:
Yes, but I would prefer to push us a bit here and try to use only --drop and targeted clean, i.e. only clean --include for instances. Or even better later, not use --drop to prefer the upsert and then targeted do drop.